streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/worker/tasks/StreamTask.java

```diff
@@ -130,7 +105,16 @@ private void prepareTask() {
 
   public void start() {
     this.thread.start();
-    LOG.info("started {}-{}", this.getClass().getSimpleName(), taskId);
+    LOG.info("Start stream task: {}-{}", this.getClass().getSimpleName(), taskId);
   }
 
+  public void close() {
+    this.running = false;
+    if (thread.isAlive() && !Ray.getRuntimeContext().isSingleProcess()) {
+      Runtime.getRuntime().halt(0);
```

Comment:
Can you explain and add comments here why we need to call `halt` and why `halt(0)` here. It's less frequently used, I think some comments helps long term maintenance a lot

Reply:
Runtime halt is proposed cause System.exist can't kill process absolutely.

Reply:
Runtime halt is proposed cause System.exist can't kill process absolutely.

Reply:
Comments have been added.